config.py

```diff
@@ -141,6 +30,22 @@ def __init__(self):
         if not os.path.exists(self.captionsFolder):
             os.makedirs(self.captionsFolder)
```

Comment:
Surround by try/catch accounting for file not found error.

Reply:
Similar to my comment below, 
This is to make a folder if it doesn't exist, like when the code is run for the very first time. Afterwards, it will run as expected. 
Is there a case that I missed where this could throw an error?

Reply:
I will resolve this.